torch/testing/_creation.py

```diff
@@ -14,12 +14,9 @@
     torch.complex64: torch.float32,
     torch.complex128: torch.float64,
 }
-float_to_corresponding_complex_type_map = {
```

Comment:
This was never used anywhere.

Reply:
Correction: this is used in two other modules. However, since we no longer have the original map, I put this into `common_dtype`.